alpine-php/builder/Makefile

```diff
@@ -1,5 +1,5 @@
-include ../../../Makefile.conf
 include ../../Makefile.conf
+include ../Makefile.conf
```

Comment:
Technically the one in one directory up is only `Makefile` not `Makefile.conf` does that Make a difference?

Reply:
Wut? Not on my source tree it's not.

Reply:
ummm. I button mashed and was just looking in the UI. :( unno.

![Screenshot 2020-11-18 at 08 06 48](https://user-images.githubusercontent.com/1299320/99502570-2a265600-2975-11eb-9e29-3ad261e078d0.png)
![Screenshot 2020-11-18 at 08 07 18](https://user-images.githubusercontent.com/1299320/99502584-2c88b000-2975-11eb-8894-f6ff63ce1e26.png)
![Screenshot 2020-11-18 at 08 07 07](https://user-images.githubusercontent.com/1299320/99502590-2e527380-2975-11eb-83c6-93982044410d.png)

Reply:
(And then if I go up one more in the directory tree it's `Makefile.conf`)

![Screenshot 2020-11-18 at 08 10 21](https://user-images.githubusercontent.com/1299320/99502828-88ebcf80-2975-11eb-915c-5542a4339e46.png)

Reply:
```
cafuego@calvados:~/Development/unocha/docker-images/alpine-php/builder (cafuego/cleanup-deprecated-images $%)
; head Makefile 
include ../../Makefile.conf
include ../Makefile.conf
cafuego@calvados:~/Development/unocha/docker-images/alpine-php/builder (cafuego/cleanup-deprecated-images $%)
; ls -la ../Makefile.conf ../../Makefile.conf 
-rw-r--r--  1 cafuego  staff  1664 18 Sep 15:36 ../../Makefile.conf
-rw-r--r--  1 cafuego  staff    10 17 Aug  2018 ../Makefile.conf
```

All as it should be.